### ISSUE TITLE
search - maps

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#9697]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

https://github.com/ArctosDB/arctos/issues/9694 was caused by Google turning off a tool (and we all somehow seem to have missed the deprecation notice, sorry!). This tool was used to draw search polygons on the map on /search. There is no straight replacement. There are a couple "rebuild everything" options (eg https://terradraw.io/) all of which are probably way "heavier" than we need, and at least one JS plugin. A super quick glance suggests the plugin is probably a little twitchy when plunked down in amongst all the complexity that's already going on (/search is a VERY dynamic environment), and I haven't actually confirmed that it can draw polygons. A roll-your-own option is possible, but doesn't sound like much fun to maintain in Google's ecosystem.

Do we need this functionality at all? (I think probably yes.) 

If so, do we need complex polygons, can we get by with something simpler, are we missing some other map-based magic? (Not a clue...)

Do we want to dig deeper into the "patch" option? (I think probably not.)

If we're still going, is Google Maps the best tool for this job? Maybe we should explore leaflet (or ??), for this, or this and everything else. (The functional needs question above might answer this one.)

What else should we be thinking about?

HELP!

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by DerekSikes** [comment]
BerkeleyMapper has a draw polygon feature... maybe their solution can be used somehow? Eg search Arctos, map all the results, draw polygon on BM to filter search, download... or something?

**Comment by dustymc** [comment]
BerkeleyMapper is using leaflet, maybe ~~stealing~~ "borrowing" code from @jdeck88 is a compelling reason to move away from Google (at some scale)

**Comment by jdeck88** [comment]
Borrow away!  Lovely to be free from Google on BerkeleyMapper now :)

**Comment by mkoo** [comment]
Berkeleymapper and its children (looking at the Geome-db.org interface) has had some good spatial searching fxns (either draw yourself or by polygon). See also it's API of intersection! #9657
